### DISCUSSION TITLE
Problem with hard limits CNC

### DISCUSSION THREAD

**Discussion by WIWOlder** [arkypita/LaserGRBL#1423] [Q&A]
When i want Run the home cycle on the GRBL after Z axis collides with limit switch ،  the software ( cnc openpilot ) locked and need to reset the app for continue auto level .
And that's why every program such a UGS gives an error and i need help guys PLSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS :((((((((((

**Comment by gmmanonymus111** [comment]
This sounds like the intended behaviour of GRBL when a limit switch is pressed.
Without proper information (software used, GRBL version, engraver type, some images maybe), we can't really help.
This issue doesn't seem to be LaserGRBL related so far.
